SingularityService/src/test/java/com/hubspot/singularity/data/SingularityValidatorTest.java

```diff
@@ -0,0 +1,63 @@
+package com.hubspot.singularity.data;
+
+import static org.mockito.Mockito.*;
```

Comment:
We generally try to stay away from any import `*` if we can. Can probably tune your IDE settings for this